model_analyzer/model_manager.py

```diff
@@ -83,6 +83,7 @@ def run_model(self, model):
                 measurement = self._metrics_manager.execute_run_config(
                     run_config)
             else:
+                logger.info("Skipping illegal run configuration")
```

Comment:
Is it possible to print anything about why it is illegal? Would probably need to enhance is_legal_combination()

Reply:
It is there already, but only shows up in verbose (it is a debug flag):
https://github.com/triton-inference-server/model_analyzer/blob/tgerdes-update-logging/model_analyzer/config/run/model_run_config.py#L100